### ISSUE TITLE
ACME simp_le has TOS mismatch

### ISSUE THREAD

**Issue by domenkozar** [NixOS/nixpkgs#31744]
```
14:19:10,160:DEBUG:requests.packages.urllib3.connectionpool:395: https://letsencrypt.org:443 "GET /documents/LE-SA-v1.2-November-15-2017.pdf HTTP/1.1" 404 123                                                                                                       Nov 15 14:19:10 staging-infra.ec2.hydra acme-hydra.iohk.io-start[7344]: 2017-11-15 14:19:10,163:DEBUG:simp_le:1237: TOS hash: f33c27745f2bd87344be790465ef984a972fd539dc83bd4f61d4242c607ef1ee                                                                                                                                                          Nov 15 14:19:10 staging-infra.ec2.hydra acme-hydra.iohk.io-start[7344]: TOS hash mismatch. Found: f33c27745f2bd87344be790465ef984a972fd539dc83bd4f61d4242c607ef1ee.                                                                                                                                                                                     Nov 15 14:19:10 staging-infra.ec2.hydra acme-hydra.iohk.io-start[7344]: Debugging tips: -v improves output verbosity. Help is available under --help.    Nov 15 14:19:10 staging-infra.ec2.hydra systemd[1]: acme-hydra.iohk.io.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
```

https://github.com/zenhack/simp_le/commit/48b406f983e21386390dc9b89ba794a48c2673ef

**Comment by domenkozar** [comment]
Fix in https://github.com/NixOS/nixpkgs/pull/31745

**Comment by fpletz** [comment]
Merged #31745.
